From 5ec4fae58d79e521d520c6c83a096c0226692078 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Wed, 19 Nov 2008 13:53:34 -0800
Subject: [PATCH 11/21] Staging: comedi: fix comedi_fc.h checkpatch.pl issues.

This resolves all the resolvable checkpatch.pl issues in the
comedi_fc.h file.

Cc: David Schleef <ds@schleef.org>
Cc: Frank Mori Hess <fmhess@users.sourceforge.net>
Cc: Ian Abbott <abbotti@mev.co.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/comedi/drivers/comedi_fc.h |   29 +++++++++++++++--------------
 1 file changed, 15 insertions(+), 14 deletions(-)

--- a/drivers/staging/comedi/drivers/comedi_fc.h
+++ b/drivers/staging/comedi/drivers/comedi_fc.h
@@ -30,28 +30,30 @@
 #include "../comedidev.h"
 
 /* Writes an array of data points to comedi's buffer */
-extern unsigned int cfc_write_array_to_buffer(comedi_subdevice * subd,
-	void *data, unsigned int num_bytes);
+extern unsigned int cfc_write_array_to_buffer(comedi_subdevice *subd,
+					      void *data,
+					      unsigned int num_bytes);
 
-static inline unsigned int cfc_write_to_buffer(comedi_subdevice * subd,
-	sampl_t data)
+static inline unsigned int cfc_write_to_buffer(comedi_subdevice *subd,
+					       sampl_t data)
 {
 	return cfc_write_array_to_buffer(subd, &data, sizeof(data));
 };
 
-static inline unsigned int cfc_write_long_to_buffer(comedi_subdevice * subd,
-	lsampl_t data)
+static inline unsigned int cfc_write_long_to_buffer(comedi_subdevice *subd,
+						    lsampl_t data)
 {
 	return cfc_write_array_to_buffer(subd, &data, sizeof(data));
 };
 
-extern unsigned int cfc_read_array_from_buffer(comedi_subdevice * subd,
-	void *data, unsigned int num_bytes);
+extern unsigned int cfc_read_array_from_buffer(comedi_subdevice *subd,
+					       void *data,
+					       unsigned int num_bytes);
 
-extern unsigned int cfc_handle_events(comedi_device * dev,
-	comedi_subdevice * subd);
+extern unsigned int cfc_handle_events(comedi_device *dev,
+				      comedi_subdevice *subd);
 
-static inline unsigned int cfc_bytes_per_scan(comedi_subdevice * subd)
+static inline unsigned int cfc_bytes_per_scan(comedi_subdevice *subd)
 {
 	int num_samples;
 	int bits_per_sample;
@@ -61,9 +63,8 @@ static inline unsigned int cfc_bytes_per
 	case COMEDI_SUBD_DO:
 	case COMEDI_SUBD_DIO:
 		bits_per_sample = 8 * bytes_per_sample(subd);
-		num_samples =
-			(subd->async->cmd.chanlist_len + bits_per_sample -
-			1) / bits_per_sample;
+		num_samples = (subd->async->cmd.chanlist_len +
+			       bits_per_sample - 1) / bits_per_sample;
 		break;
 	default:
 		num_samples = subd->async->cmd.chanlist_len;
